shellfoundry_traffic/test_helpers.py

```diff
@@ -6,13 +6,14 @@
 from typing import Optional, Tuple
 
 import yaml
-import xml.etree.ElementTree as ET
+import xml.etree.ElementTree as Element_Tree
 
 from cloudshell.api.cloudshell_api import (CloudShellAPISession, ResourceAttributesUpdateRequest, AttributeNameValue,
                                            ResourceInfo, CreateReservationResponseInfo, SetConnectorRequest)
 from cloudshell.helpers.scripts.cloudshell_dev_helpers import attach_to_cloudshell_as
 from cloudshell.shell.core.session.cloudshell_session import CloudShellSessionContext
-from cloudshell.shell.core.driver_context import (ResourceCommandContext, ResourceContextDetails, AutoLoadCommandContext,
+from cloudshell.shell.core.driver_context import (ResourceCommandContext, ResourceContextDetails,
+                                                  AutoLoadCommandContext,
                                                   ConnectivityContext, InitCommandContext, AppContext)
```

Comment:
Please try to push as much into each line, the last line should be the shortest.
Something like:
from cloudshell.shell.core.driver_context import (ResourceCommandContext, ResourceContextDetails,
                                                  AutoLoadCommandContext, ConnectivityContext, InitCommandContext,
                                                  AppContext)

Reply:
Fixed.